app/assets/javascripts/directives/views/historyMenu.ts

```diff
@@ -69,17 +73,20 @@ class HistoryMenuCtrl extends PureViewCtrl implements HistoryScope {
   }
 
   async openRemoteRevision(revision: RemoteHistoryListEntry) {
-    const itemUuid = this.item.uuid;
     this.fetchingRemoteHistory = true;
-    revision = await this.application.historyManager!.fetchRemoteRevision(itemUuid, revision);
+    const remoteRevision = await this.application.historyManager!.fetchRemoteRevision(this.item.uuid, revision);
     this.fetchingRemoteHistory = false;
+    if (!remoteRevision) {
+      this.application.alertService!.alert("The remote revision could not be loaded. Please try again later.");
```

Comment:
We should alert the user in case the revision can not be loaded for some reason.